ts/components/PinScreenComponent.tsx

```diff
@@ -1,57 +1,53 @@
+import I18n from "i18n-js";
```

Comment:
Do we need to maintain the word "Screen" in the name?

Reply:
I think it's useful to understand that this component is used only by the couple: `OnboardingPinScreen` and `PinScreen`.

Do you have any further ideas?

Reply:
My point it:
- 1 or 2 is just a particular case of N
- It's not responsibility of the component to specify where it is used.

Reply:
renamed in [6cc93af](https://github.com/pagopa/io-app/pull/4303/commits/6cc93af310f46ce03c9fcbe3994d80d394befbcc)